nix/options.nix

```diff
@@ -55,6 +55,14 @@ in
       '';
     };
 
+    deployment.forwardSSHAgent = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        This option whether to forward the local SSH agent socket to the remote.
```

Comment:
Let's explain why this is maybe a bad idea.

Reply:
This is now removed and `sshOptions` should be used instead. Now it's up to the user to know why a certain option is a bad idea.